app/workers/validate_pull_request_worker.rb

```diff
@@ -10,7 +10,9 @@ def self.perform_async(id)
 
     queue = Sidekiq::Queue.new('default')
     queue.each do |job|
-      if (job.args.sort == this_args.sort) && (job.item['class'] == name)
+      # use tally. the arrays can contain strings and ints. we cannot call sort on heterogenous arrays
+      # see also https://twitter.com/BastelsBlog/status/1311421975827546112
+      if (job.args.tally.to_a - this_args.tally.to_a).empty? && (job.item['class'] == name)
```

Comment:
I would call `this_args.tally.to_a` outside of the loop to avoid doing it for every job.